viewing_party/party.py

```diff
@@ -1,23 +1,165 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {}

```

Comment:
Instead of creating an empty dictionary, you can build a dictionary literal an example could look like
```python
if title and genre and rating:
        movie =  {
            "title": title,
            "genre": genre,
            "rating": rating
            }
        return movie
```